lib/rules/i18n-ellipsis.js

```diff
@@ -24,16 +25,13 @@ var rule = module.exports = function( context ) {
 			}
 
 			node.arguments.forEach( function( arg ) {
-				if ( 'Literal' !== arg.type || 'string' !== typeof arg.value ) {
-					return;
-				}
-
-				if ( -1 !== arg.value.indexOf( '...' ) ) {
+				let argumentString = getTextContentFromNode( arg );
```

Comment:
I found CircleCI had been silently failing on tests and this had failed to be flagged as an issue. Our `package.json` Node supports at the time of this pull request was `0.10.0`, where `let` is not a valid keyword.

- Fixed silent failing tests in d06a13fb4e8650994e1182c3c39c4a346c6f025e
- Replaced `let` with `var` in 6e2a654d3a7f101a78caa6e704eb2d4403a2fc48
- Bumped Node minimum to 0.12.0 in 8698df573578800ed388191af6f8fa0ee4481f41 
  - Because `0.10.0` doesn't satisfy our [`requireindex` dependency's own minimum](https://github.com/stephenhandley/requireindex/blob/b6cb37fc11923e6bc36a3db29b70e90b82c91fbb/package.json#L26))